packages/vite/src/node/optimizer/index.ts

```diff
@@ -673,6 +673,28 @@ export function runOptimizeDeps(
                 browserHash: metadata.browserHash,
               })
             }
+          } else {
+            // workaround Firefox warning by removing blank source map reference
+            // https://github.com/evanw/esbuild/issues/3945
+            const output = meta.outputs[o]
+            // filter by exact bytes
+            if (output.bytes === 93) {
+              const jsMapPath = path.resolve(
+                processingCacheDirOutputPath,
+                path.relative(processingCacheDirOutputPath, o),
+              )
```

Comment:
I'm a bit confused of the relative and resolve of the same variable here. Could this be this instead:
```suggestion
              const jsMapPath = path.resolve(o)
```